addon/components/dropdown.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+import layout from '../templates/components/dropdown';
+import calculatePosition from 'ember-basic-dropdown/utils/calculate-position';
+
+export default Component.extend({
+  layout,
+  classNames: 'cs-component-dropdown',
+  selected: null,
+  multiple: false,
+  labelText: 'Select an option',
+
+  calculatePosition(trigger) {
+    let position = calculatePosition(...arguments);
+    let { left, width, top } = position.style;
+    left -= 1;
+    width += 2;
+
+    if (!trigger.classList.contains('ember-power-select-multiple-trigger')) {
+      top -= 32;
+      position.style.top = top;
+    }
+    position.style.left = left;
+    position.style.width = width;
+
+    return position;
+  }
```

Comment:
The `power-select` and `power-select-multiple` (input for multiple select renders inline) components render the input in different places, so we need to reposition the single select input. There's also a 1px misalignment that this fixes

<img width="718" alt="Screen Shot 2019-05-02 at 10 10 16 AM" src="https://user-images.githubusercontent.com/118005/57093404-ab01fc00-6cc2-11e9-94b0-44fc99516d4b.png">
<img width="722" alt="Screen Shot 2019-05-02 at 10 10 07 AM" src="https://user-images.githubusercontent.com/118005/57093442-c0772600-6cc2-11e9-9787-f3883203c1a6.png">


<img width="130" alt="Screen Shot 2019-05-02 at 10 11 03 AM" src="https://user-images.githubusercontent.com/118005/57093422-b3f2cd80-6cc2-11e9-81c8-7f085747fcde.png">

Reply:
Is there not a way to do this using only CSS? You could specify `renderInPlace=true` as an option to `{{power-select}}` which renders the dropdown next to the trigger in the DOM (instead of using `ember-wormhole`), which _should_ allow you to just override the CSS.

Reply:
The issue is the way that the markup is structured. The border on the top box is a parent of the options with the countries. The other thing that this function does is to help make the multi and single select inputs consistent. With power-select the inputs are in different places.

Reply:
you could try wrapping the `{{power-select}}` and `{{power-select-multiple}}` in the template with `<div>`s with different classes. But I agree that we should punt on it until we have more concrete designs.